creator/extract_configs/templates/participant_observations_config.py

```diff
@@ -0,0 +1,29 @@
+"""
+This is an extract configuration for a Participant Observations file.
+
+See: https://www.notion.so/d3b/a7c4a98cab854d05aaf3e8e73469ac57?v=b5ee7a0282084f25b6b0a2a925e1bec6
+
+Required Columns:
+    Participant ID
+    Age at Onset
+    Age at Onset Units
+    Observation Name
+    Verification Status
+
+Optional Columns:
+    MONDO Code
+    NCIt Code
+    ICD10 Code
+    SNOMED CT
+"""
+from kf_lib_data_ingest.common.concept_schema import CONCEPT
+from kf_lib_data_ingest.etl.extract.operations import keep_map
+
+source_data_url = "{{ download_url }}"
+
+
+operations = [
+    keep_map(in_col="Participant ID", out_col=CONCEPT.PARTICIPANT.ID),
+    keep_map(in_col="Age at Onset", out_col=CONCEPT.DIAGNOSIS.EVENT_AGE_DAYS),
+    keep_map(in_col="Observation Name", out_col=CONCEPT.PHENOTYPE.NAME),
```

Comment:
kids first phenotypes must be observed positive or negative.

Reply:
Does this mean that we should consider all entries as positive, or require an additional field to annotate it?

Reply:
We have explicitly non-positive entries for "I looked and they definitely don't have X" so either it needs an annotation or we stop doing that.